### PR TITLE
Update impact page and navbar icon

### PR DESCRIPTION
**Pull request by kerry918**
- Updated the testimonials and growth stories with permissions.

### CODE REVIEW

**Comment by keyualiceli** [comment]
This looks good! 

the only suggestions i have is that on the landing page upcoming/current events tabs, we can use the same formatting and font as the styles we use in the initiatives event tabs, to make it more consistent. This is what it looks like to me rn
image
image

File: src/components/initiatives/PastEvents.js [hunk 1]
```diff
@@ -94,29 +94,29 @@ export default function PastEvents() {
...
                 </Grid>
                 <Grid item xs={9}>
                   <Box>
                       <Typography gutterBottom variant="h5" component="h2">
-                        Summer 2020 Mentorship Program
+                        Spring 2020 Mentorship Program
                       </Typography>
                       <Typography variant="body2" color="textSecondary" component="p">
-                        Add a description of the project
+                        66 Mentor/Mentee Pairs
```
Comment (eyhhuang) [inline src/components/initiatives/PastEvents.js:112]: Can we make it "Matched 66 Mentor/Mentee Pairs"

File: src/components/initiatives/PastEvents.js [hunk 2]
```diff
@@ -94,29 +94,29 @@ export default function PastEvents() {
                         Fall 2020 Mentorship Program
                       </Typography>
                       <Typography variant="body2" color="textSecondary" component="p">
-                        Add a description of the project
+                        110 Mentor/Mentee Pairs
```
Comment (eyhhuang) [inline src/components/initiatives/PastEvents.js:97]: "Matched 110 Mentor/Mentee Pairs"

File: src/components/initiatives/PastEvents.js [hunk 3]
```diff
@@ -143,12 +144,13 @@ export default function PastEvents() {
                         Study Abroad Experience Sharing - June ‘20
                       </Typography>
                       <Typography variant="body2" color="textSecondary" component="p">
-                        Add a description of the project
+                          Learn about the experiences shared by eight current and past
```
Comment (eyhhuang) [inline src/components/initiatives/PastEvents.js:147]: I think the descriptions for "Work Abroad Experience Sharing - July ‘20" and "Study Abroad Experience Sharing - June ‘20" should be more consistent. 

E.g. both should use "UW students" or "students from the University of Waterloo"

and maybe the word abroad one can be: "Learn about the experiences of current and past UW students have have interned outside of Canada and the U.S. and how they found those opportunities"

File: src/components/initiatives/PastEvents.js [hunk 4]
```diff
@@ -181,12 +184,12 @@ export default function PastEvents() {
                               Resume Critique - Sept ‘20
                             </Typography>
                             <Typography variant="body2" color="textSecondary" component="p">
-                              Add a description of the project
+                              246 total resume reviews
```
Comment (eyhhuang) [inline src/components/initiatives/PastEvents.js:187]: "Facilitated 264 resume reviews"

File: src/components/landing_page/Event.js [hunk 5]
```diff
@@ -12,20 +12,20 @@ function EventTab(props){
...
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Coffee Chats are an opportunity for students to connect with multiple experienced upper year students to explore more areas of technology. (107 Coffee Chat pairs)
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
-                <h1 className={classes.past1}>Spring 2020 Coffee Chat Program</h1>
+                <h1 className={classes.past1}>Fall 2020 Mentorship Program</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Mentorship program is an ongoing program to pair first year with upper year mentors to provide guidance. (110 Mentor/Mentee pairs)
```
Comment (eyhhuang) [inline src/components/landing_page/Event.js:22]:
```suggestion
                    Mentorship is an ongoing program that pairs lower year students with upper year mentors to provide guidance. (110 Mentor/Mentee pairs)  
```
is the 110 pairs in total or just for this term?

File: src/components/landing_page/Event.js [hunk 6]
```diff
@@ -12,20 +12,20 @@ function EventTab(props){
     if (props.index === 0){
         return(
             <div>
-                <h1 className={classes.past1}>Winter 2020 Mentorship Program</h1>
+                <h1 className={classes.past1}>Spring 2020 Coffee Chat Program</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Coffee Chats are an opportunity for students to connect with multiple experienced upper year students to explore more areas of technology. (107 Coffee Chat pairs)
```
Comment (eyhhuang) [inline src/components/landing_page/Event.js:17]: 107 just for this term? might be better if we can get a total number of coffee chats

File: src/components/landing_page/Event.js [hunk 7]
```diff
@@ -34,18 +34,18 @@ function EventTab(props){
...
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Coffee Chats are an opportunity for students to connect with multiple experienced upper year students to explore more areas of technology.
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
                 <h1 className={classes.past1}>Fall 2020 Mock Interviews</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Practice your interview skills with upper-year students who have scored co-ops at amazing companies, big and small!
```
Comment (eyhhuang) [inline src/components/landing_page/Event.js:42]:
```suggestion
                    Practice and refine interview skills with upper-year students who have scored co-ops at amazing companies, big and small!
```

File: src/components/landing_page/Event.js [hunk 8]
```diff
@@ -34,18 +34,18 @@ function EventTab(props){
...
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
                 <h1 className={classes.past1}>Winter 2021 Waterloo Coop Intro</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Introduction to Waterloo coop structure and tips in how to be prepared and apply for jobs.
```
Comment (eyhhuang) [inline src/components/landing_page/Event.js:47]:
```suggestion
                    Introduction to Waterloo co-op structure and tips in how to be prepared and apply for jobs. 
```
Should make sure all usages of "co-op" are consistent.

**Review by eyhhuang — COMMENTED** [review]
Looks pretty good so far! Just left some comments about the project descriptions

**Comment by eyhhuang** [comment]
Is it possible to make the testimonials a horizontal scroll with arrows (so the same as growth stories) instead of clicking on different points in tihs line?

Maybe I'm using it incorrectly lol but it's a little hard for me to swipe to see different testimonials

image

**Pull request merged**